tests/test_pymediainfo.py

```diff
@@ -112,6 +117,33 @@ def test_parse_non_existent_path_pathlib(self):
         path = self.pathlib.Path(data_dir) / "this file does not exist"
         self.assertRaises(FileNotFoundError, MediaInfo.parse, path)
 
+
+class MediaInfoTestFilenameTypes(unittest.TestCase):
+    def test_convert_filename_str(self):
```

Comment:
You used the same name for all three methods so only the last one is actually run.